cassdegrees/templates/managecourses.html

```diff
@@ -1,148 +1,76 @@
-                                <input class="text tfull" id="text" type="text" name="code" minlength="8" maxlength="8" size="8" aria-required="true" required>
-                            </p>
-
-                            <p>
-                                <label for="text">Year {{ required }}</label>
-                                <input class="text tfull" id="number" type="number" name="year" min="1990" aria-required="true" required>
-                            </p>
 
-                            <p>
-                                <label for="text">Semester(s)</label>
-                                <select class="text tfull" name="semesters[]" size="2" multiple>
-                                    <option value="semester1">Semester 1</option>
-                                    <option value="semester2">Semester 2</option>
-                                </select>
-                            </p>
-
-                             <p>
-                                <label for="text">Units {{ required }}</label>
-                                 <!-- max value is chosen arbitrarily -->
-                                <input class="text tfull" id="number" type="number" name="units" min="0" step="6" aria-required="true" required>
-                            </p>
-
-                            <p>
-                                <label for="text">Requisite Course(s)</label>
-                                <select class="text tfull" name="requisites[]" size="5" multiple>
-                                    {% for course in courses %}
-                                        <option value="{{ course.code }}">{{ course.code }}</option>
-                                    {% endfor %}
-                                </select>
-                            </p>
-
-                            <p>
-                                <label for="text">Incompatible Course(s)</label>
-                                <select class="text tfull" name="incompatibles[]" size="5" multiple>
-                                    {% for course in courses %}
-                                        <option value="{{ course.code }}">{{ course.code }}</option>
-                                    {% endfor %}
-                                </select>
-                            </p>
-                        </fieldset>
-                    {% endwith %}
-
-                    <p class="text-right">
-                        <input class="btn-uni-grad btn-large" type ="submit" value="{{ action }}">
+    <div id="{{ action }}Page" {% if action == 'Delete' or render.hide_form %}hidden{% endif %}>
+        <form class="anuform" action="/manage_courses/?action={{ action }}" method="post">
+            {% csrf_token %}
+            {# Allow manage_courses() function to behave appropriately based on origin of request. The post request can either come from list.html or managecourses.html #}
```

Comment:
IMO, this needs to not point to the origin url itself (there is a Referer header anyway, isn't there?), but as to the specific functionality that navigating from this page brings.

Reply:
What are your suggested changes? I'm not sure how to proceed from your comment.

Reply:
Instead of `managecourses.html`, name this option what it actually *changes*. If we are hardcoding the names of webpages here, this will get messy quick.

Reply:
Yeah OK that's a good reason. Will do.